packages/cozy-konnector-libs/src/libs/saveBills.js

```diff
@@ -197,4 +203,68 @@ function checkRequiredAttributes(entries) {
     }
   }
 }
+
+function appendContractsDatas(tempEntries, options) {
+  if (options.contractLabel && options.contractId === undefined) {
+    log('warn', 'contractLabel used without contractId')
+    return tempEntries
+  }
+  let newEntries = tempEntries
+  if (options.contractId) {
+    // if contractId passed by option
+    newEntries = appendContractDatasByOption(newEntries, options)
+  } else if (billsHaveContractId(newEntries)) {
+    // if contractId passed by bill attribute
+    newEntries = appendContractDatasByAttribute(newEntries, options)
+  }
+  return newEntries
+}
+
+function appendContractDatasByOption(newEntries, options) {
+  // Surcharge label if not define
```

Comment:
```suggestion
  // Override label with contractId if not defined
```

Reply:
I found a better comment:
// Define contractlabel from contractId if not set in options